packages/opentelemetry-web/src/utils.ts

```diff
@@ -127,6 +127,11 @@ export function getResource(
   >(),
   initiatorType?: string
 ): PerformanceResourceTimingInfo {
+  // de-relativize the URL before usage (does no harm to absolute URLs)
+  const a = document.createElement('a');
```

Comment:
Nit: can you move this line in the beginning of file so that the element is created once only and then reused.

Reply:
Yes, I just ran the experiment and we can cache the 'a'; sorry for not running it earlier.  I'll fix this momentarily.